docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -534,17 +534,42 @@ Migrating Ansible content to a different collection
 
 First, look at `Ansible Collection Checklist <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_.
 
-To migrate content from one collection to another, you need to create three PRs as follows:
+To migrate content from one collection to another, if the collections are parts of Ansible distribution:
```

Comment:
Which is defined in https://github.com/ansible-community/ansible-build-data/blob/main/2.10/ansible.in

Reply:
good suggestion! added